.yarn/patches/tsconfig-paths-npm-3.14.2-90ce75420d.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/try-path.js b/lib/try-path.js
+index de11ccbf9090658eb0ee61bdba1984ee21bcb3b7..2e75f72483aa7b05bf0a2cde06b3c009eb2103e8 100644
+--- a/lib/try-path.js
++++ b/lib/try-path.js
+@@ -85,6 +85,6 @@ function matchStar(pattern, search) {
+     if (search.substr(search.length - part2.length) !== part2) {
+         return undefined;
+     }
+-    return search.substr(star, search.length - part2.length);
++    return search.substr(star, search.length - part2.length - part1.length);
```

Comment:
We need to try and upstream this. Also we need to determine if it is a bug before merging 😓

Reply:
Can we try opening a well-reasoned PR with tests etc that is an easy merge for the maintainers?

Reply:
Sure, but do you think that should block merging this?

Reply:
It would be nice to get confirmation that it's a bug, but probably not blocking